InitializeWithLocalBinaryDependencies

tsavo-at-pieces opened the pull request.

tsavo-at-pieces committed:
Add download_libs.sh script

tsavo-at-pieces committed:
Final Testing

tsavo-at-pieces committed:
All tests passing

tsavo-at-pieces committed:
Unit tests and readme

brian-at-pieces commented:
Everything's looking good but should we consider making some of these hardcoded values dynamic? Primarily the hashes and lib versions

tsavo-at-pieces committed:
Update local_assets_readme.md

tsavo-at-pieces committed:
Update .gitignore

tsavo-at-pieces committed:
Throw path not found error

tsavo-at-pieces committed:
Support both os level folder hierarchy or hash only hierarchy for the passed dylibs dir

i.e.
macos/c4f61c9bde1085be63f32dd54ca8829e/
OR
/c4f61c9bde1085be63f32dd54ca8829e/

windows/c2ddf39c36bd6ab58d86b27ddc102286
OR just
/c2ddf39c36bd6ab58d86b27ddc102286

..same on linux

linux/6af4f73a0a0e59cb7e1a272a9fa0828a/

or
/6af4f73a0a0e59cb7e1a272a9fa0828a
